### PR TITLE
Fix formatting on wiki pages according to ASC

### LINKED ISSUE #1338: Correct pages from updated ASC
~Opened this, so people don't waste time. I'll be doing this.~

~Doing the originally planned idea. A commit for every minor change or fix. **Edit: Might be per PR.**
(i.e. See the first referenced PR that I closed. It'll have that format.)~

~Tagging outdated for articles will come last.~

### PR DESCRIPTION
**Pull request by Joehuu**
### Description

Leaving this here because I might be missing some files that need to be fixed. And there will be more formatting commits later on. ~Also en.md is the priority, but I plan to edit all the files even if they are outdated.~

Edit: Scratch that, adding outdated tags to the outdated translations. Will not edit them.

Edit: Whitespace edits are automatic from my editor, so I can't help that. It needs to be removed anyway.

### Status

- [X] WIP
- [X] Pending Review

### Additional Notes 

Resolves #1338.

Not doing these reworks, edits, tournaments, etc. (Basically open PRs)
- Probably all tournaments (Maybe separate PR for faster merge)
- ASC (PR in progress)

### CODE REVIEW

File: wiki/History_of_osu!/2011/en.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 ## December
 
-The osu!wiki was announced, allowing users to add all sorts of information about everything osu! related.
+The osu! wiki was announced, allowing users to add all sorts of information about everything osu! related.
 
 Links:
 
-- [osu!wiki project](https://osu.ppy.sh/community/forums/topics/68479)
+- [osu! wiki project](https://osu.ppy.sh/community/forums/topics/68479)
```
Comment (MegaApplePi) [inline wiki/History_of_osu!/2011/en.md]: this is the name of the thread, leave it as is

File: wiki/redirect.yaml [hunk 2]
```diff
@@ -488,7 +488,7 @@
 "hoo/2016":                     "History_of_osu!/2016"
 "hoo/2017":                     "History_of_osu!/2017"
 "hoo/2018":                     "History_of_osu!/2018"
-"hoo/osu!wiki":                 "History_of_osu!/osu!_wiki"
+"hoo/osu!_wiki":                "History_of_osu!/osu!_wiki"
```
Comment (MegaApplePi) [inline wiki/redirect.yaml]: do not change this, leave it as is

**Comment by Ephemeralis** [comment]
meeeeeerrrrrge conflictssss

**Comment by Joehuu** [comment]
Redoing the edits to be all around the wiki.

**Pull request closed without merging**